lib/restforce/db/worker.rb

```diff
@@ -104,11 +113,7 @@ def track
         if tracker
           runtime = Time.now
 
-          if tracker.last_run
-            log "SYNCHRONIZING from #{tracker.last_run.iso8601}"
-          else
-            log "SYNCHRONIZING"
-          end
+          log "SYNCHRONIZING#{" from #{tracker.last_run.iso8601}" if tracker.last_run}"
```

Comment:
I found some meaningful places to reduce the amount of code in the class; should be good for another look. Thanks for keeping me honest.